src/main/java/seedu/address/model/todolist/Type.java

```diff
@@ -0,0 +1,13 @@
+package seedu.address.model.todolist;
+
+/**
+ * Represents the type of the task.
+ */
+public enum Type {
```

Comment:
Perhaps you could extend a similar concept further and allow users to add tags to a particular task? The tagging feature seems to have some similarities with the Type enumeration here.

Reply:
Oh yes, I just realized that Tag can probably replace Type.